gpiozero/output_devices.py

```diff
@@ -837,29 +838,36 @@ class Motor(SourceMixin, CompositeDevice):
         The GPIO pin that the backward input of the motor driver chip is
         connected to.
 
+    :param int enable:
+        (Optional) The GPIO pin that enables the motor. Required for some motor
+        controller boards. Defaults to ``None``.
+
     :param bool pwm:
         If ``True`` (the default), construct :class:`PWMOutputDevice`
         instances for the motor controller pins, allowing both direction and
         variable speed control. If ``False``, construct
-        :class:`DigitalOutputDevice` instances, allowing only direction
+        :class:`OutputDevice` instances, allowing only direction
```

Comment:
Looks like this should have remained `DigitalOutputDevice` rather than being changed to `OutputDevice` ?
(since the code does `PinClass = PWMOutputDevice if pwm else DigitalOutputDevice`)